math/netgen/files/patch-CMakeLists.txt.diff

```diff
@@ -8,7 +8,7 @@ index 0a05ebb1..677d586b 100644
  
  set(NG_INSTALL_SUFFIX netgen CACHE STRING "Suffix appended to install directories (project name)")
  
--if(APPLE)
+-if(APPLE AND NOT EMSCRIPTEN)
 +if(APPLE AND FALSE)
```

Comment:
Isn't X && FALSE always FALSE?

Reply:
@barracuda156 ?

Reply:
@pmetzger I think I merely adapted the patch we had. Can be changed.

Reply:
@pmetzger Updated to use just `FALSE`.